doc/src/Examples.dox

```diff
@@ -143,7 +143,7 @@
  * context.retention = context.kTransient
  *
  * # We can now resolve a token we may have if it is a reference.
- * resolved_asset = manager.resolveEntityReference(some_string, context)
+ * resolved_assets = manager.resolveEntityReference([some_string], context)
```

Comment:
[fix] I think the subscript change is better here as otherwise it may be interpreted that you may get several assets back from `some_string`. We know we only ever put a single sting in, so it could only ever logically be `resolved_asset`.

Reply:
Updated in 2f7a20c